releases.md

```diff
@@ -11,15 +11,15 @@ The following table lists in-support .NET releases.
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-5/) | [Preview][policies] | [6.0 Preview 6][6.0 Preview 6] | November, 2024 |
-| [.NET 5](release-notes/5.0/README.md) | [November 10, 2020](https://devblogs.microsoft.com/dotnet/announcing-net-5-0/) | [Current][policies] | [5.0.8][5.0.8] | May, 2022 |
-| [.NET Core 3.1](release-notes/3.1/README.md) | [December 3, 2019](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/) | [LTS][policies] | [3.1.17][3.1.17] | December 3, 2022 |
-| [.NET Core 2.1](release-notes/2.1/README.md) | [May 30, 2018](https://devblogs.microsoft.com/dotnet/announcing-net-core-2-1/) | [LTS][policies] | [2.1.28][2.1.28] | [August 21, 2021](https://devblogs.microsoft.com/dotnet/net-core-2-1-will-reach-end-of-support-on-august-21-2021/) |
-
-[6.0 Preview 6]: release-notes/6.0/preview/6.0.0-preview.6.md
-[5.0.8]: release-notes/5.0/5.0.8/5.0.8.md
-[3.1.17]: release-notes/3.1/3.1.17/3.1.17.md
-[2.1.28]: release-notes/2.1/2.1.28/2.1.28.md
+| [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-7/) | [Preview][policies] | [6.0 Preview 7][6.0 Preview 7] | November, 2024 |
```

Comment:
is it correct to list .NET 6 here as a supported version while it's still on preview? It's not officially supported yet until we get to the go-live licenses.

Reply:
@richlander for his input here.

Reply:
We should come up with a better model for that for .NET 7. It doesn't matter much for .NET 6 at this point. Agree with the concern.